src/main/java/org/jitsi/jicofo/bridge/Bridge.java

```diff
@@ -348,4 +302,10 @@ public String toString()
                      getRelayId(),
                      getRegion());
     }
+
+    public int getLastReportedPacketRatePps()
```

Comment:
How is it this doesn't seem to get called by the selection code?  Or anywhere exception for logging/stats?

Reply:
Becaus `compareTo` uses the field directly. I can fix if you like

Reply:
Gotcha, I see it now.  I think it's fine as-is.